examples/community/lpw_stable_diffusion_xl.py

```diff
@@ -1107,12 +1107,14 @@ def __call__(
         #     negative_pooled_prompt_embeds=negative_pooled_prompt_embeds,
         #     lora_scale=text_encoder_lora_scale,
         # )
```

Comment:
Could we maybe delete the commented code?

Reply:
Commented code removed, and use `black` to fix the code quality problem.